samples/app/src/main/java/com/sourcepoint/app/v6/di/AppModule.kt

```diff
@@ -71,7 +72,7 @@ val appModule = module {
                 messLanguage = MessageLanguage.ENGLISH
                 campaignsEnv = CampaignsEnv.PUBLIC
                 +(CampaignType.GDPR)
-                +(CampaignType.CCPA to listOf(("location" to "US")))
+//                +(CampaignType.CCPA to listOf(("location" to "US")))
```

Comment:
Is this comment expected to be in `develop`? Pls remove a comment or uncomment a line or add an explanation

Reply:
Another one that slipped, thanks for catching it.